server/utils/comments/comments.util.js

```diff
@@ -16,6 +16,17 @@ class commentValidation {
     }
     return next();
   }
+
+  static validateCommentType(req, res, next) {
+    const { commentType } = req.body;
+    if (commentType) {
```

Comment:
The code on line `22` can be refactored by checking for if the commentType is not in the request body. This will make the code look neat.

Reply:
The comment Type is optional. That's why it's in a `if block`.

Reply:
Ok.